lib/servers/serialserver/serial_server.py

```diff
@@ -151,10 +151,10 @@ def baudrate(self, c, data=None):
         """Sets the baudrate."""
         ser = self.getPort(c)
         if data is None:
-            return long(ser.getBaudrate())
-        else: 
```

Comment:
what happened to the function getBaudrate()? is it used somewhere else and what did it different than just using ser.baudrate?